docs/builds/guides/integration/configuration.md

```diff
@@ -133,6 +133,50 @@ ClassicEditor
 	} );
 ```
 
+### Extended format
+
+You can use the extended toolbar configuration format to access additional options:
+
+```js
+    toolbar: {
+        items: [ 'bold', 'italic', '|', 'undo', 'redo', '-', 'numberedList', 'bulletedList' ],
+        viewportTopOffset: 30,
+        shouldNotGroupWhenFull: true
+    }
+```
+
+ * **`items`** &ndash; An array of toolbar item names. Most of the components (buttons, dropdowns, etc.) which can be used as toolbar items are described under {@link features/index Features} tab. Full list is defined in {@link module:ui/componentfactory~ComponentFactory editor.ui.componentFactory} and can be listed using the following snippet: `Array.from( editor.ui.componentFactory.names() )`. More details could be found in {@link framework/guides/creating-simple-plugin Creating a simple plugin}.
+
+ * **`viewportTopOffset`** &ndash; The offset (in pixels) from the top of the viewport used when positioning a sticky toolbar.
+ Useful when a page with which the editor is being integrated has some other sticky or fixed elements
+ (e.g. the top menu). Thanks to setting the toolbar offset, the toolbar will not be positioned underneath or above the page's UI.
+
+ * **`shouldNotGroupWhenFull`** &ndash; When set to `true`, the toolbar will stop grouping items and let them wrap to the next line when there is not enough space to display them in a single row. This setting is `false` by default, what enables items grouping.
+
+#### Separating toolbar items
+
+You can use `'|'` to create a separator between groups of toolbar items. Works in both config formats:
+
+```js
+    toolbar: [ 'bold', 'italic', '|', 'undo', 'redo' ]
+```
+
+```js
+    toolbar: {
+        items: [ 'bold', 'italic', '|', 'undo', 'redo' ]
+    }
+```
+
+There are also two ways of arranging toolbar items into multiple lines. Both require the extended format.
+
+1. Set `shouldNotGroupWhenFull` to true, so items will not be grouped when the toolbar overflows but will wrap to the new line instead.
```

Comment:
Should these two options be the same? This kind of looks confusing.

Reply:
Not sure if I understand your point. Are you thinking about changing writing about "two ways" and those two points into something like: "use shouldNotGroupWhenFull to enable automatic wrapping and add '-' if you want an explicit break"? It could sound better.

Reply:
I mean "Set to true gives you option one" and "set to true gives you option two" might look like there is some error. So rewriting it the way you suggest should do the trick. I know this is opposed to another configuration option and a careful reader will notice it, still, this caught my eye.

Reply:
Corrected, please check it now.

Reply:
Looks nice!